service/src/main/java/bio/terra/workspace/service/resource/controlled/cloud/azure/storageContainer/VerifyAzureStorageContainerCanBeCreatedStep.java

```diff
@@ -27,22 +35,29 @@
  * CreateAzureStorageContainerStep} to ensure idempotency of the create operation.
  */
 public class VerifyAzureStorageContainerCanBeCreatedStep implements Step {
-
+  public static final String AZURE_STORAGE_ACCOUNT_RESOURCE_TYPE =
```

Comment:
Maybe factor out these strings into a separate class of static strings?
I know this one is used more than once...

Reply:
It turns out that it's used only in one place in dispatcher above. Forgot to remove unused code. This constant is unused.